infra/gcp/terraform/k8s-infra-elections/main.tf

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This file defines:
+- Google Project k8s-infra-elections to host the elections platform
+- Google services enabled for the project
+- GCP internal IP for the database k8s-infra-elections-db
+*/
+
+locals {
+  project_id = "k8s-infra-elections"
```

Comment:
Why put this in its own project / module vs. adding more to the public project / module? Ostensibly you're provisioning a resource / service that an app on the public cluster is going to connect to.

Reply:
This is a great question. I think I got triggered by the term "elections" and I wanted to apply some specific security measures because I don't think `elekto` should be trusted to run `aaa` but I realize now that the other workloads are at the same level of trust.